src/careamics/lightning/callbacks/prediction_writer_callback/file_path_utils.py

```diff
@@ -0,0 +1,56 @@
+"""Module containing file path utilities for `WriteStrategy` to use."""
+
+from pathlib import Path
+from typing import Union
+
+from careamics.dataset import IterablePredDataset, IterableTiledPredDataset
+
+
+# TODO: move to datasets package ?
+def get_sample_file_path(
+    ds: Union[IterableTiledPredDataset, IterablePredDataset], sample_id: int
```

Comment:
A there-is-a-single-speck-of-dust-on-the-book-shelf type of comment, in general I prefer to avoid very short variable names! (which probably I am guilty of often...).

Here we know what `ds` is obviously, but it would still be nicer to read to have `dataset`. 

You can ignore that comment!

Reply:
Ok sure I will update, I agree definitely in function signatures variables should be more verbose.